ommx_dwave_adapter/adapter.py

```diff
@@ -127,13 +134,18 @@ def solve(
         """
         return cls.sample(
             ommx_instance, token=token, time_limit=time_limit, label=label
-        ).best_feasible()
+        ).best_feasible
 
     @property
     def sampler_input(self) -> ConstrainedQuadraticModel:
         """The dimod.ConstrainedQuadraticModel representing this OMMX instance"""
         return self.model
 
+    @property
+    def solver_input(self) -> ConstrainedQuadraticModel:
+        """The dimod.ConstrainedQuadraticModel representing this OMMX instance"""
+        return self.sampler_input
```

Comment:
```suggestion
        return self.model
```

余計なメソッド呼び出しを減らせるのでSuggestionのように変更したほうが良いと考えます。